lib/reporting/reports/suppliers/helpers/line_items_access_helper.rb

```diff
@@ -38,8 +38,11 @@ def suppliers_adjustments(line_item, adjustment_type = 'EnterpriseFee')
           end
 
           def adjustments_by_type(line_item, type, included: false)
+            is_tax = type == :tax
+            return 0.0 if is_tax && !supplier(line_item).charges_sales_tax
```

Comment:
It seems strange that the report needs to check if `charges_sales_tax`. I thought that the application logic should be able to work this out. I see that `Spree::TaxRate` checks for this in `match`, but I don't think that helps here.

Oh well, it turns out it's quite common to refer to this directly in other reports, all good 👍